### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by S1SYPHOS** [berteh/ScribusGenerator#149]
Hey there,
I'd like to propose a new version release!
Well, that's it ..

Cheers!

PS: Is there an easy way to replace variables throughout a `sla` file (basically XML, right?) - could you point me to the line being responsible for that in your code?

**Comment by berteh** [comment]
new release with no new functionnality ?

I don't quite understand your question about replacing variables.

If you want to rename one variable, or replace it everywhere with some given text I suggest you use a simple Python script such as https://github.com/berteh/ScribusGenerator/blob/master/utils/ConvertVAR_NEXT-RECORDToSG28.py  (this one replaces all instances of VAR_NEXT-RECORD with SG_NEXT-RECORD)

You can use the full power of regular expressions in the substitution if needed.

If that was not your question kindly explain it again to me, maybe with an example ?

hope this helps.
Berteh.

**Comment by S1SYPHOS** [comment]
Hey there.
Well, could as well be a patch release, since there were some fixes made in the meantime (16 commits isn't nothing).

I solved the (variable replacement) issue I had by simply using this:

```python
def replace(file_name, pattern, value=''):
    file = fileinput.input(file_name, inplace=True)
    for line in file:
        replacement = value
        line = re.sub(pattern, replacement, line)
        sys.stdout.write(line)
    file.close()
``` 
.. and threw it at some `.sla` files - still learning to embrace python, remember .. but alas, our project now includes generation of `.eml` files to send to publishers to let them know we included their books in our [list of recommended novelties](https://github.com/fundevogel/recommended-novelties), (nearly) all in python :fox_face: 

There's still some PHP code left (mainly historical reasons), and there's nothing to be gained by porting everything to python.
